pages/ugm2023.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: UGM 2023
+---
+
+## When, Where, and How
+27-29 September 2023
+
+Faculty of Sciences of the University of Lisbon (Lisbon, Portugal)
+
+Live, in-person format
+
+## What to Expect
+Whether you are a new MDAnalysis contributor, an MDAnalysis core developer, or a researcher using (or interested in using) MDAnalysis for your work in academia or industry, this UGM is for you. You can expect:
+* Keynote talks from experts in the molecular dynamics simulations space
+* Talks related to developing and using MDAnalysis for diverse scientific applications
+* Tutorials diving into new features
+* Direct access to the MDAnalysis core developer team to help shape future development of MDAnalysis
+* Networking opportunities to foster new collaborations
+* Community events, including a hackathon and social outings
+
+## Program and Keynotes
+A draft agenda and keynote speaker(s) will be announced soon.
+
+## Registration and Call for Proposals
+Registration and a call for proposals will open soon.
+
+## Pricing
+The event will be free to attend for all. 
```

Comment:
See above... do you really want to make it free? With no "skin in the game" people may just register and never show up and take up seats.

Reply:
this is a "save the date" announcement. let's keep info to a minimum and add details about pricing, abstract submission and keynotes later after we've discussed internally one more time..

Reply:
Done - I kept the headers for abstracts/keynotes as placeholders though